docs/how-to/deploy/deploy-aws.md

```diff
@@ -121,18 +119,18 @@ For more guidance on sizing production environments, see the [Requirements page]
 We also recommend to deploy a [Data Integrator](https://charmhub.io/data-integrator) for creating an admin user to manage the content of the Kafka cluster:
 
 ```shell
-juju deploy data-integrator admin \
+juju deploy data-integrator \
   --config extra-user-roles=admin \
-  --config topic-name=admin-topic
+  --config topic-name=__admin-user
 ```
 
 And integrate it with the Kafka application:
 
 ```shell
-juju integrate kafka admin
+juju integrate kafka data-integrator
 ```
 
-For more information on Data Integrator and how to use it, please refer to the [how-to manage applications](how-to-manage-applications) guide.
+For more information on Data Integrator and how to use it, please refer to the [how-to manage applications](how-to-client-connections) guide.
```

Comment:
Suggestion: Please make sure you update the links text too.

```suggestion
For more information on Data Integrator and how to use it, please refer to the [how-to manage client connections](how-to-client-connections) guide.
```

Reply:
Yes, I did. It looks like this:

```suggestion
For more information on Data Integrator and how to use it, please refer to the [](how-to-client-connections) guide.
```

It should use the closest next heading (after the anchor) for the link's text. But I haven't tested it yet thoroughly. Make sure to try it out before committing.